src/api.ts

```diff
@@ -0,0 +1,10 @@
+export const URLPosts = 'https://mate-academy.github.io/react_dynamic-list-of-posts/api/posts.json';
+export const URLUsers = 'https://mate-academy.github.io/react_dynamic-list-of-posts/api/users.json';
+export const URLComments = 'https://mate-academy.github.io/react_dynamic-list-of-posts/api/comments.json';
+
+export const fetchData = async <T>(url: string): Promise<T[]> => {
+  const response = await fetch(url);
+  const data = response.json();
```

Comment:
```suggestion
  const data = await response.json();
```
если будет много данных то у тебя будет ошибка без await